GEOSaana_GridComp/GSI_GridComp/CMakeLists.txt

```diff
@@ -700,6 +701,8 @@ if (CMAKE_Fortran_COMPILER_ID MATCHES Intel)
             set_source_files_properties (${src} PROPERTIES COMPILE_FLAGS "")
          elseif (${src} MATCHES prepbykx.f)
             set_source_files_properties (${src} PROPERTIES COMPILE_FLAGS "${GEOS_Fortran_Release_Flags} ${common_Fortran_flags} ${BYTERECLEN} ${EXTENDED_SOURCE} ${FP_MODEL_STRICT} ${ALIGNCOM}")
+         elseif (${src} MATCHES prepbysaid.f)
+            set_source_files_properties (${src} PROPERTIES COMPILE_FLAGS "${GEOS_Fortran_Release_Flags} ${common_Fortran_flags} ${BYTERECLEN} ${EXTENDED_SOURCE} ${FP_MODEL_STRICT} ${ALIGNCOM}")
```

Comment:
Why do you want to force the release flags?    Would be nice to be able to use debug flags when intended?

Reply:
Hi Tom,

I have simply copied from the code above "prepbykx.f". Not sure about the nuances in the CMakefile. Please let me know what I should change it to.

-Mohar

Reply:
@mohar123 and @tclune I will approave this and resolve the issue w/ the flag elsewhere.